### PR TITLE
Fixes to let counter example compile

### PR DESCRIPTION
**Pull request by RomanBaiocco**
On the main branch, if I cd into `Example/counter` and run `swift build` I am unable to compile the plugin due to some errors. These changes fix those errors.

### CODE REVIEW

File: Examples/counter/Sources/counter/Actions/DecrementAction.swift
Comment (emorydunn) [inline Examples/counter/Sources/counter/Actions/DecrementAction.swift:1]: Remove the `controllers` and `encoder` properties and instead conform the class to `KeyAction`.

File: Examples/counter/Sources/counter/Actions/IncrementAction.swift
Comment (emorydunn) [inline Examples/counter/Sources/counter/Actions/IncrementAction.swift:1]: Remove the `controllers` and `encoder` properties and instead conform the class to `KeyAction`.

**Review by emorydunn — CHANGES_REQUESTED** [review]
Thanks for fixing the package manifest. I've added some comments to the actions.

**Comment by RomanBaiocco** [comment]
Just tried again! Let me know if I did it incorrectly, this is literally the first thing I've done after getting the basics of the language down 😅

**Review by emorydunn — APPROVED** [review]

**Pull request merged**